_includes/tutorials/joining-stream-stream/ksql/markup/dev/transient-join.adoc

```diff
@@ -0,0 +1,11 @@
+Let's cross both streams to come up with a new stream that contains more insightful information such as which orders have been shipped, which warehouse shipped them, and how long did it take to ship each order. The following query does a inner join between the orders stream and the shipments stream, and specifies a window duration of seven days to represent the SLA of one week to ship orders. This will block and continue to return results until it's limit is reached or you tell it to stop.
```

Comment:
- Why did we use an inner join? Roughly what does it do?
- The 7 day window isn't really an SLA -- we should be more specific and talk about how it controls for the buffering space to allow joins.

Reply:
I think something that's really crucial to mention somewhere is *why* a stream/stream join? Why not a stream/table join here? This is something people really get tripped up on.

Reply:
One other thing to note is that the window is using event time. This bit can be a black box when people look at s/s joins.